src/chi/components/steps/steps.scss

```diff
@@ -59,33 +61,33 @@ $breakpoints: ('sm', 'md', 'lg', 'xl');
     }
 
     & > a {
-      color: $brand-color;
-      font-weight: 600;
+      font-weight: $font-weight-semibold;
       margin-left: 0.5rem;
       white-space: nowrap;
     }
 
     & > .chi-steps__line {
       border-color: $brand-color;
+      margin: 0.5rem 0.75rem;
     }
 
-    &.-active {
+    &.-completed {
```

Comment:
Looks great but we're missing the progress arrows illustrated in the design. Should be able to construct via an after pseudo element on `& > .chi-steps__line` within `&.-completed`. The simplest solution might be to just render an icon-chevron-right `content: '\e920';` to give the illusion of an arrow.

Reply:
Built with pseudo-element but using border-top and border-right instead of an icon. Icon was. not rendering as expected